wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/NewsFragment.java

```
@@ -96,7 +96,9 @@ public void requestComplete(final JSONObject json, final boolean isCache) {
                         handleNews(json, handler);
                     }
                 });
-        task.startDownload(this);
+        try {
+            task.startDownload(this);
+        } catch (WiGLEAuthException waex) {} //unauthenticated call
```

Empty catches always wig me out, can we at least have an info log?